app/Livewire/Server/ValidateAndInstall.php

```
@@ -25,6 +25,8 @@ class ValidateAndInstall extends Component
 
     public $supported_os_type = null;
 
+    public $prerequisites_installed = null;
```

PHPMD (2.15.0)

Avoid excessively long variable names like $prerequisites_installed. Keep variable name length under 20. (undefined)

(LongVariable)